Phonebook.Frontend/src/app/app.component.ts

```diff
@@ -154,7 +158,7 @@ export class AppComponent implements OnInit, OnDestroy {
           description: 'Message for just set no cookie url',
           id: 'PageInformationNewUrlNoCookies',
           value:
-            'Save the site URL as a favourite now in order to not get any more startup-dialogs. Please notice: You won\'t get any information about updates or releases with the set url parameter.'
+            "Save the site URL as a favourite now in order to not get any more startup-dialogs. Please notice: You won't get any information about updates or releases with the set url parameter."
```

Comment:
Can you find the according setting in prettier to use single quotes instead of doubles and save it?

Reply:
That's because of the qutoe and the "/" for escaping the quote in "won't". If I save the file it automaticly replaces single quotes with double quotes and removes the "/". After that it says that single qoute should be used. Strange behaviour!

Reply:
Can you use `` ` `` then?

Reply:
I did, in the latest commit.